recipes/taskipy/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "taskipy" %}
+{% set version = "1.10.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0006429f708f530fc7add28ca51fd41f6c6e1fbe86763ff05125e1af3e8bdc7e
+
+build:
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - task = taskipy.cli:main
+
+requirements:
+  host:
+    - python
+    - pip
+    - poetry
+  run:
+    - python
+    - pip
+    - tomli >=1.2.3  # [py==36]
+    - tomli >=2.0.1  # [py>=37]
```

Conda-forge no longer supports py36 and, making this change, we can keep it noarch and build it only once for all Python versions.

```suggestion
    - tomli >=2.0.1
```